src/main/webapp/script.js

```diff
@@ -0,0 +1,100 @@
+var zoomedIn = false;
+var aqLayerOn = true;
+var routeVisible = false;
+
+function changeBackgroundImage(imagePath) {
+    document.body.style.backgroundImage = "url('" + imagePath + "')";
```

Comment:
Perhaps [template strings](https://developers.google.com/web/updates/2015/01/ES6-Template-Strings) would be more readable:
```
= `url('${imagePath}')`
```

(The use the the back-ticks to quote template strings is an acceptable variation to the consistent string quoting rules.)